From a83061dc2b1ed5c98aa98d0d6c064b684a272b83 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 18 Mar 2018 14:53:28 +0100
Subject: [PATCH 1/1] video: exynos: remove redundant assignments

No need to initialize variables if the next usage is an assignment.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
v2
	add missing semicolon
---
 drivers/video/exynos/exynos_dp.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/video/exynos/exynos_dp.c b/drivers/video/exynos/exynos_dp.c
index 30e4020686..2f724ee745 100644
--- a/drivers/video/exynos/exynos_dp.c
+++ b/drivers/video/exynos/exynos_dp.c
@@ -321,7 +321,7 @@ static unsigned int exynos_dp_link_start(struct exynos_dp *regs,
 
 static unsigned int exynos_dp_training_pattern_dis(struct exynos_dp *regs)
 {
-	unsigned int ret = EXYNOS_DP_SUCCESS;
+	unsigned int ret;
 
 	exynos_dp_set_training_pattern(regs, DP_NONE);
 
@@ -339,7 +339,7 @@ static unsigned int exynos_dp_enable_rx_to_enhanced_mode(
 		struct exynos_dp *regs, unsigned char enable)
 {
 	unsigned char data;
-	unsigned int ret = EXYNOS_DP_SUCCESS;
+	unsigned int ret;
 
 	ret = exynos_dp_read_byte_from_dpcd(regs, DPCD_LANE_COUNT_SET,
 					    &data);
@@ -366,7 +366,7 @@ static unsigned int exynos_dp_enable_rx_to_enhanced_mode(
 static unsigned int exynos_dp_set_enhanced_mode(struct exynos_dp *regs,
 						unsigned char enhance_mode)
 {
-	unsigned int ret = EXYNOS_DP_SUCCESS;
+	unsigned int ret;
 
 	ret = exynos_dp_enable_rx_to_enhanced_mode(regs, enhance_mode);
 	if (ret != EXYNOS_DP_SUCCESS) {
@@ -416,7 +416,7 @@ static int exynos_dp_read_dpcd_lane_stat(struct exynos_dp *regs,
 static unsigned int exynos_dp_read_dpcd_adj_req(struct exynos_dp *regs,
 		unsigned char lane_num, unsigned char *sw, unsigned char *em)
 {
-	unsigned int ret = EXYNOS_DP_SUCCESS;
+	unsigned int ret;
 	unsigned char buf;
 	unsigned int dpcd_addr;
 	unsigned char shift_val[DP_LANE_CNT_4] = {0, 4, 0, 4};
@@ -484,7 +484,7 @@ static int exynos_dp_reduce_link_rate(struct exynos_dp *regs,
 static unsigned int exynos_dp_process_clock_recovery(struct exynos_dp *regs,
 					struct exynos_dp_priv *priv)
 {
-	unsigned int ret = EXYNOS_DP_SUCCESS;
+	unsigned int ret;
 	unsigned char lane_stat;
 	unsigned char lt_ctl_val[DP_LANE_CNT_4] = {0, };
 	unsigned int i;
@@ -594,7 +594,7 @@ static unsigned int exynos_dp_process_clock_recovery(struct exynos_dp *regs,
 static unsigned int exynos_dp_process_equalizer_training(
 		struct exynos_dp *regs, struct exynos_dp_priv *priv)
 {
-	unsigned int ret = EXYNOS_DP_SUCCESS;
+	unsigned int ret;
 	unsigned char lane_stat, adj_req_sw, adj_req_em, i;
 	unsigned char lt_ctl_val[DP_LANE_CNT_4] = {0,};
 	unsigned char interlane_aligned = 0;
-- 
2.16.2

